src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -84,6 +95,26 @@
     margin-top: var(--pf-c-modal-box--m-align-top--MarginTop);
   }
 
+  &.pf-m-error {
+    --pf-c-modal-box__title-icon--Color: var(--pf-c-modal-box--m-error__title-icon--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-modal-box__title-icon--Color: var(--pf-c-modal-box--m-warning__title-icon--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-modal-box__title-icon--Color: var(--pf-c-modal-box--m-success__title-icon--Color);
+  }
+
+  &.pf-m-default {
```

Comment:
do you need a modifier for default variation?

Reply:
I think so. Without a modifier, the icon is just a regular text color icon. With an alert modifier, it represents the alert state, and "default" is an alert state. WDYT?